SECURITY.md

```diff
@@ -26,12 +26,14 @@ involving the following steps:
 ## Comments on this Policy
 
 If you have suggestions on how this process could be improved please submit 
-a ticket, open a [Discorse](https://discourse.osc.edu/) topic or open a pull request.
+a ticket, open a [Discourse](https://discourse.openondemand.org/) topic or open a pull request.
 
 ## Security Audits
 
-[Trusted CI](https://trustedci.org/), the NSF Cybersecurity Center of 
+, the NSF Cybersecurity Center of 
```

Comment:
This sentence starts with a comma - I think there's some mistake here in the way you want to reword this sentence.

Reply:
Oops.. I meant to delete that entire paragraph and replace it with the subsequent one.